rocAL_pybind/rocal_pybind.cpp

```diff
@@ -379,6 +380,7 @@ PYBIND11_MODULE(rocal_pybind, m) {
         .value("NCHW", ROCAL_NCHW)
         .value("NFHWC", ROCAL_NFHWC)
         .value("NFCHW", ROCAL_NFCHW)
+        .value("NONE", ROCAL_NONE)
```

Comment:
Let us not include None as layou outside, please check other related comments

Reply:
Removed